### PR TITLE
Update lb-s3-bucket/aws to v0.16.0 added option to force ssl on buckets

### PR DESCRIPTION
**Pull request by joshmello**
## what

Figured I would update this to use v0.16 of the lb s3 bucket module similar to the [this](https://github.com/cloudposse/terraform-aws-alb)

Adds an optional input variable to force SSL on the s3 bucket for security compliance.

## why

[PCI.S3.5] S3 buckets should require requests to use Secure Socket Layer

## references

[AWS Config](https://docs.aws.amazon.com/config/latest/developerguide/s3-bucket-ssl-requests-only.html)

### CODE REVIEW

File: .github/auto-release.yml [hunk 1]
```diff
@@ -17,7 +17,6 @@ version-resolver:
     - 'bugfix'
     - 'bug'
     - 'hotfix'
-    - 'no-release'
```
Comment (joshmello) [inline .github/auto-release.yml:20]: I am unsure why rebuilding the readme removed this. Is it needed?

File: .github/renovate.json [hunk 2]
```diff
@@ -4,9 +4,9 @@
     ":preserveSemverRanges"
   ],
   "labels": ["auto-update"],
+  "dependencyDashboardAutoclose": true,
```
Comment (joshmello) [inline .github/renovate.json:7]: Looks like this one was added. Is it needed and correct?

**Comment by goruha** [comment]
@joshmello 
Applied your changes in https://github.com/cloudposse/terraform-aws-nlb/pull/27

Closing the PR

**Pull request closed without merging**